### DISCUSSION TITLE
Converting a multipage PDF into a single image (TIFF or PNG)

### DISCUSSION THREAD

**Discussion by stevio-stack** [dlemstra/Magick.NET#1036] [Help]
You have no example for converting a multipage PDF into a single (multipage) image (TIFF or PNG). This would seem to be the most obvious choice. I am referring to your document here: https://github.com/dlemstra/Magick.NET/blob/main/docs/ConvertPDF.md
I did try AppendHorizontally and AppendVertically but neither of those is what I am after. They both produce single page output but tiny and distorted.

**Comment by dlemstra** [comment]
What are you trying to accomplish? Why does the `AppendVertically` method not work for you?

**Reply by stevio-stack** [reply]
Thank you dlemstra. I think I resolved this. I was trying to do the impossible. It works fine as a TIFF. I don't think it is possible to make a single multipage PNG.
